eventdiscovery-sdk/src/main/res/layout/schedjoules_event_list.xml

```diff
@@ -12,9 +12,27 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 app:layoutManager="LinearLayoutManager"
+                android:layout_marginBottom="20dp"
                 tools:context=".framework.eventlist.EventListActivity"
                 tools:listitem="@layout/schedjoules_list_item_event"/>
 
         <include layout="@layout/sticky_header_layout"/>
+
+
+        <com.schedjoules.eventdiscovery.framework.microfragments.eventdetails.fragments.views.SchedJoulesFooterView
```

Comment:
I think the duplication should be resolved with the footer in the details view, either with layout of its own and `include` tag (extra tags can be added to `include`), or extracted style, but probably `include` is better, imo.